### PR TITLE
Update README with OS-specific env setup instructions

### PR DESCRIPTION
**Pull request by wamartin-aml**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -129,6 +129,43 @@ If you intend to run repository tests:
 pip install -r requirements.txt
 ```
 
+
+<details><summary><strong><em>On Windows: </strong></em></summary>
```
Comment (imatiach-msft) [inline README.md]: this is good for now, but I prefer the style in the lightgbm documentation where a whole section is devoted just to one OS as opposed to this interleaved style

Comment (imatiach-msft) [reply README.md]: (it's much easier to follow)

Comment (wamartin-aml) [reply README.md]: The problem is that most of it is the same, it's just these small sections

Comment (bethz) [reply README.md]: where's pip install interpret-community?

**Pull request merged**